mysql-test/suite/tianmu/r/delete.result

```diff
@@ -226,10 +226,6 @@ INSERT INTO t1 VALUES(10),(20);
 CREATE TABLE t2(b INTEGER)ENGINE=TIANMU;
 INSERT INTO t2 VALUES(10),(20);
 SET SESSION sql_safe_updates=1;
-EXPLAIN DELETE t2 FROM t1 JOIN t2 WHERE t1.a = 10;
```

Comment:
Why do u remove these cases?

Reply:
The query execution plan is being modified. The query plan cannot be used as the check standard